README.md

```diff
@@ -59,5 +59,5 @@ npm run test
 
 ### Requirements
 
-- [Node 16 or higher](https://nodejs.org/en/download/)
-- [NPM 8 or higher](https://www.npmjs.com/package/npm)
+- [Node 20 or higher](https://nodejs.org/en/download/)
+- [NPM 9 or higher](https://www.npmjs.com/package/npm)
```

Comment:
Lts version is npm 10

Reply:
True @susnux. However, the requirements in the package.json are ^9.0.0. Even for other packages like @nextcloud/files, @nextcloud/l10n, @nextcloud/logger, @nextcloud/auth...

We will have a EBADENGINE error thrown with packages relying on this one, like @nextcloud/auth for example. Maybe we should : 
- align to ^9.0.0 in the package.json
- update all the other librairies

Reply:
> EBADENGINE error

This is only considered a warning, installing and compiling still works.

---

Our supported engines are always the LTS version which is Node 20 + NPM 10. So any other library that has other values set should update their package.json